syz-ci/jobs.go

```diff
@@ -377,7 +378,10 @@ func (jp *JobProcessor) bisect(job *Job, mgrcfg *mgrconfig.Config) error {
 	}
 	trace := new(bytes.Buffer)
 	cfg := &bisect.Config{
-		Trace:    io.MultiWriter(trace, log.VerboseWriter(3)),
+		Trace: &debugtracer.GenericTracer{
+			TraceWriter: io.MultiWriter(trace, log.VerboseWriter(3)),
+			OutDir:      "",
```

Comment:
I think there is something missing here.
We don't set OutDir, but set DebugDir below. And tools/syz-bisect/bisect.go sets OutDir.
I think we need to set OutDir here and make pkg/bisect use debugtracer to store debug files and remove config.DebugDir.

Reply:
This is now fixed in latest version.